CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
already in rt-cubed main CMakeLists.txt

Reply:
You must be compiling differently…. If I run cmake on this CMakeLists.txt after cloning the repo, cmake issues warnings about project and min version not being set.  This (and package()) quells those messages.  Is there some other manner of compilation being expected or reason to not support running cmake on the checkout?

Reply:
Sadeep wrote a readme for the build process: https://github.com/BRL-CAD/arbalest/blob/master/README.md (just updated it with actual repositories) Of course, patches must conform with it.

An independent arbalest project is preferable to the current setup, sure. But, this needs some additional work, e.g. the C++ library must be found. The good new is, that there is already a proof of concept with MOOSE and BrlCadGUI. I don't however want to confuse Himanshu with it. That's why, I would like to postpone it until his project ended.